<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerApp</class>
 <widget class="QWidget" name="SchedulerApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ежедневник</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="eventNameLineEdit">
     <property name="placeholderText">
      <string>Введите название события</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>hh:mm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="eventListWidget"/>
   </item>
  </layout>
 </widget>
</ui>
